rocky/tests/conftest.py

```diff
@@ -609,3 +621,127 @@ def mock_scheduler_client_task_list(mocker):
     mock_scheduler_client_session.get.return_value = response
 
     return mock_scheduler_client_session
+
+
+@pytest.fixture
+def valid_time():
+    return datetime.now(timezone.utc)
+
+
+@pytest.fixture
+def octopoes_api_connector(request) -> OctopoesAPIConnector:
```

Comment:
I think it would be better to move the fixtures that are only for the integration tests to `tests/integration/conftest.py`.